RHEL6_7/networking/iptables/check

```diff
@@ -55,8 +55,8 @@ if service_is_enabled "iptables" || service_is_enabled "ip6tables"; then
     cp $POST_SCRIPT $POSTUPGRADE_DIR/$SCRIPT_NAME
   fi
 
-  log_warning "Read the remediation instructions."
-  exit_informational
+  log_slight_risk "Read the remediation instructions."
```

Comment:
The message is too generic. It should be changed to something like:
- *The iptables or ip6tables service is enabled. Read the remediation instructions.*

What do you think?

Reply:
Yes, I will add that info